.circleci/config.yml

```diff
@@ -100,6 +100,18 @@ jobs:
         paths:
         - images/
 
+
+  build-windows:
```

Comment:
This would need to be included in `workflows` above, if you want it to run each time.
It's probably faster just to add the `GOOS=windows` step to `build` after `save_cache`.